src/main/java/org/generationcp/middleware/api/brapi/StudyServiceBrapiImpl.java

```diff
@@ -317,7 +318,7 @@ public long countStudyInstances(final StudySearchFilter studySearchFilter) {
 		final LocationSearchRequest request = new LocationSearchRequest();
```

Comment:
@lik-83 seems like the request object is not needed

Reply:
Done